crates/sncast/src/helpers/scarb_utils.rs

```diff
@@ -51,6 +52,11 @@ impl Default for CastConfig {
     }
 }
 
+pub struct BuildConfig {
+    pub scarb_toml_path: Utf8PathBuf,
+    pub json: bool,
+}
+
```

Comment:
[[Nitpick]] can we move this struct above the only function that uses it?

Reply:
Sure, done in newest commit :+1: